helpers/addresses.py

```diff
@@ -681,6 +683,13 @@
         "badger_wbtc_bunni_token_309720_332580": "0x849535B729768321ADeD4EdE1aCEfa03a5f96FDE",
         "badger_wbtc_bunni_gauge_309720_332580": "0xd614e32088C5f5Dc389014Ec5727E03a0811b8c8",
     },
+    "liquis": {
+        "booster": "0x631e58246A88c3957763e1469cb52f93BC1dDCF2",
+        "liquislit_staking": "0xB96Bce10480d2a8eb2995Ee4f04a70d48997856a",
```

Comment:
if possible, could we rename this variable to `crvDepositor` to make it more clearly according to Liquis docs

https://github.com/liquisfi/deployments/blob/main/contracts.mainnet.json

Reply:
their naming is due to be a fork of another existing codebase, i think for us to provide better context to stick with the nomenclature of the tokens of liquis ecosystem imo